__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data/index");
+
+beforeEach(() => seed({ articleData, commentData, topicData, userData }));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  it("GET: 200 responds with an array of all topics to the client", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { rows } }) => {
```

Comment:
lets not name it rows. Lets be descriptive with our names by naming the data here topics